SConstruct

```diff
@@ -1626,26 +1646,59 @@ if env['matlab_toolbox'] == 'y':
 # /usr/local because of dist-packages vs site-packages
 env['debian'] = any(name.endswith('dist-packages') for name in sys.path)
 
+# Check whether Cantera should be installed into a conda environment
+if conda_prefix is not None:
+    if env["layout"] != "conda":
+        # identify options selected either on command line or in cantera.conf
+        selected_options = set(line.split("=")[0].strip() for line in cantera_conf)
+        use_conda = not selected_options & \
+            {"layout", "prefix", "python_prefix", "python_cmd"}
```

Comment:
Can you group this operation with parentheses instead of the backslash continuation? Otherwise, the second set could become its own variable for clarity.